src/main/java/uk/gov/hmcts/reform/cpo/ApplicationParams.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.cpo;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Named
+@Singleton
+@Getter
+public class ApplicationParams {
+
+    @Value("${ccd.defaultPageSize}")
```

Comment:
this is not a CCD microservice
```suggestion
    @Value("${cpo.defaultPageSize}")
```

Reply:
DONE